compass/landice/tests/ismip6_forcing/ismip6.cfg

```diff
@@ -0,0 +1,51 @@
+# config options for ismip6 antarctic ice sheet data set
+[ismip6_ais]
```

Comment:
This file needs to be called `ismip6_forcing.cfg` to be included automatically.  If you want to keep the current name, you will need to add it explicitly to each test case in its `configure()` method.

Reply:
When I don't supply a user config file, I'm seeing:
```
Traceback (most recent call last):
  File "/home/ac.xylar/anvil/miniconda3/envs/compass_ismip6_forcing/bin/compass", line 33, in <module>
    sys.exit(load_entry_point('compass', 'console_scripts', 'compass')())
  File "/gpfs/fs1/home/ac.xylar/mpas-work/compass/add_ismip6_forcing_testgroup/compass/__main__.py", line 63, in main
    commands[args.command]()
  File "/gpfs/fs1/home/ac.xylar/mpas-work/compass/add_ismip6_forcing_testgroup/compass/setup.py", line 372, in main
    setup_cases(tests=tests, numbers=args.case_num,
  File "/gpfs/fs1/home/ac.xylar/mpas-work/compass/add_ismip6_forcing_testgroup/compass/setup.py", line 146, in setup_cases
    setup_case(path, test_case, config_file, machine, work_dir,
  File "/gpfs/fs1/home/ac.xylar/mpas-work/compass/add_ismip6_forcing_testgroup/compass/setup.py", line 244, in setup_case
    test_case.configure()
  File "/gpfs/fs1/home/ac.xylar/mpas-work/compass/add_ismip6_forcing_testgroup/compass/landice/tests/ismip6_forcing/atmosphere/__init__.py", line 34, in configure
    base_path_ismip6 = self.config.get(section="ismip6_ais",
  File "/home/ac.xylar/anvil/miniconda3/envs/compass_ismip6_forcing/lib/python3.9/site-packages/mpas_tools/config.py", line 111, in get
    return self.combined.get(section, option)
  File "/home/ac.xylar/anvil/miniconda3/envs/compass_ismip6_forcing/lib/python3.9/configparser.py", line 781, in get
    d = self._unify_values(section, vars)
  File "/home/ac.xylar/anvil/miniconda3/envs/compass_ismip6_forcing/lib/python3.9/configparser.py", line 1152, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'ismip6_ais'
```
which clued me in to this being missing.

Reply:
I don't see `output_base_path` here.  The default behavior seems to be to write things to a local path called `None`.